### ISSUE TITLE
Cross domain errors

### ISSUE THREAD

**Issue by ujovlado** [keboola/kbc-ui#3617]
Povedal by som, ze niektore z nich su realne. Vcera sa mi to stalo v eu-central-1.

Zakladam to narychlo sem, no oprava sa bude asi tykat Syrupu a suvisiacich komponent.

```
darth-vader:~ vlado$ http https://syrup.keboola.com/provisioning
HTTP/1.1 404 Not Found
Access-Control-Allow-Headers: *
Access-Control-Allow-Methods: *
Access-Control-Allow-Origin: *
Cache-Control: must-revalidate, no-cache, no-store, private
Connection: keep-alive
Content-Length: 150
Content-Type: application/json
Date: Wed, 14 Aug 2019 06:21:53 GMT
Server: Apache/2.4.25 (Debian)
X-Powered-By: PHP/5.6.40

{
    "code": 404,
    "error": "User error",
    "exceptionId": "provisioning-1be12318bab497e0b894f3a4154f4bbe",
    "message": "Route not found",
    "runId": 0,
    "status": "error"
}

darth-vader:~ vlado$
```

Problemy moze sposobovat:

```
Access-Control-Allow-Headers: *
Access-Control-Allow-Methods: *
```

**Comment by Halama** [comment]
Při jaký akci se ti to stalo?

**Comment by ujovlado** [comment]
Bol som na stranke Sandboxov a hral som sa tym. Padali mi requesty na zmazanie sandboxu a pod.

Vsetko kvoli cross origin. A myslim, ze za to mozu tie hviezdicky `*` pri Headers a Methods.

**Comment by Halama** [comment]
Zkouším to a jede mi to ok. Zkoušel jsem i předtím teda...
Vypadá to že ale každí metoda ty hlavičky vrací trochu jinak.

![image](https://user-images.githubusercontent.com/903531/63005034-032cca80-be7c-11e9-8b4e-02caeecf3238.png)

![image](https://user-images.githubusercontent.com/903531/63005055-0d4ec900-be7c-11e9-8d63-6a7317d7a03a.png)

![image](https://user-images.githubusercontent.com/903531/63005072-150e6d80-be7c-11e9-8fc9-6407259c4795.png)

**Comment by Halama** [comment]
Asi by to ale nemělo souviset s tou dockerizací ne?

**Comment by ujovlado** [comment]
Nie, nesuvisi to s dockerizaciou, tam sme sa hlaviciek nechytali.

Bude sa ti to diat ale len na browseroch ako tu:
http://status.keboola.com/error-when-saving-configuration-post-mortem
